Sensor sequence note

CCOSTAN opened the pull request:
Added additional note about adding sensors in sequence.

fabaff commented:
This circumstance is mentioned clearly at the beginning of the [devices](https://home-assistant.io/getting-started/devices.html) page and affects most if not all elements in the configuration file. I think that we should not add this to selected pages because from my point of view this leads to confusion.

CCOSTAN commented:
I agree except I keep fielding the same question in Chat. For whatever reason, people aren't finding it.  But Totally up to you guys.  I added it to a bunch of places but I also added it to the main device page thinking it would have resolved most queries.

Maybe all sensor pages should have a link back to the main device page instead?

fabaff commented:
Sorry, I moved to gitter and you to github...

The sample configuration.yaml file shows an example, too. Well, on the [devices](https://home-assistant.io/getting-started/devices.html) it looks a bit like that this format is needed for sensors only. Perhaps we should update the order.
